### PR TITLE
Page title

### PR DESCRIPTION
**Pull request by JC4github**
### Changes

Describe the Pull Request here. Add any references and info to help reviewers understand your changes.

- Changed page title and logo

### Tests applied

What testing has been applied to validate changes are accurate and correct

- visual inspection 

### Issue ticket number(s)

Enter the issue numbers resolved by this pull request

- 37

### Checklist

- [x] Documented
- [x] Linting tests successful
- [x] merge updated prod branch into development branch

### CODE REVIEW

File: .husky/.gitignore
Comment (jaskfla) [inline .husky/.gitignore:1]: Any chance you could put the `.husky/` file back?

We can reconfigure the pre-commit hooks later, as it seems we have at least two people having trouble. But at least for now it remains part of the recommended project setup.

File: index.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/typefaceoff_logo.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>TypeFaceOff</title>
```
Comment (jaskfla) [inline index.html]: Seeing as we’ve now decided on an “official” capitalisation, might get you to update this.

Sorry about the busywork 😅

File: typefaceoff_logo.svg
Comment (jaskfla) [inline typefaceoff_logo.svg:1]: Nice!

**Review by jaskfla — CHANGES_REQUESTED** [review]
Just the capitalisation of the title, and perhaps reinstating Husky files.

Otherwise looks good!

File: .husky/.gitignore
Comment (JC4github) [reply .husky/.gitignore:1]: the .husky/ files are in my project on VScode, I'm not sure why they are not added to the commit.

File: .husky/pre-commit
Comment (jaskfla) [inline .husky/pre-commit:1]:
```suggestion
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

npx lint-staged
```

**Review by jaskfla — COMMENTED** [review]
~~Looks like the `pre-commit` file you put back was empty.~~

~~Otherwise~~ looks good to merge 🥳

**Review by jaskfla — APPROVED** [review]

**Pull request merged**